tox.ini

```diff
@@ -9,7 +9,7 @@ deps =
 commands =
     {envpython} --version
     flake8 cartoframes tests
-    py.test --basetemp="{envtmpdir}" tests/unit
+    # py.test --basetemp="{envtmpdir}" tests/unit
```

Comment:
Why is this?

Reply:
This was because unit tests were not updated. Fixed in `bq-mvt` branch.